rosbridge_library/test/internal/test_message_conversion.py

```diff
@@ -55,6 +46,17 @@ def do_primitive_test(self, data_value, msgtype):
                 self.assertEqual(msg["data"], msg2["data"])
                 self.assertEqual(msg2["data"], inst.data)
 
+    def do_byte_test(self, data_value, msgtype):
+        for msg in [{"data": data_value}]:
+            inst = ros_loader.get_message_instance(msgtype)
+            c.populate_instance(msg, inst)
+            self.assertEqual(inst.data, bytes([data_value]))
```

Comment:
I'm not sure this is right. std_msgs/Byte holds a single byte, why should it be a `bytes` instance?

Reply:
I'm not so familiar with this, but as far as I know, it's because `rclpy` defines so.

![image](https://user-images.githubusercontent.com/31987104/137632994-05d78b1a-f9bb-467c-b403-796c1dac84b9.png)

https://design.ros2.org/articles/generated_interfaces_python.html

```python
In [1]: from example_interfaces.msg import Byte                                                                                                                                                                    

In [2]: Byte().data                                                                                                                                                                                                
Out[2]: b'\x00'

In [3]: Byte().data == bytes([0])                                                                                                                                                                                  
Out[3]: True

In [4]: Byte().data                                                                                                                                                                                                
Out[4]: b'\x00'
```